j2v/generation/result_writer.py

```diff
@@ -28,8 +28,7 @@ def get_view_str(self, views_dimensions_expr):
             source_table = ""
             if view == self.table_alias:
                 source_table = """\n  sql_table_name: {sql_table} ;;""".format(sql_table=self.sql_table_name)
-
-            views_out.append(lt.view_start_str_template.format(name=view, base_table=source_table))
+            views_out.append(lt.view_start_str_template.format(name=view.lower(), base_table=source_table))
```

Comment:
In case you have upper case array in sample JSON, this will break things, views are referenced also in Joins in the explore file.
Example to check:
{
  "apiVersion": "v3.4",
  "data Provider": "Eat me",
  "RESTAURANTS": [
    {
      "name": "Super Burger",
      "city": "Sydney",
      "country": "Australia",
      "address": "Big Street 3",
      "currency": "AUD",
      "menu": [
        {
          "dishName": "BurgerPlus",
          "price": 10,
          "ingredients": ["Meat", "Cheese", "Bun"]
        }
      ]
    }
  ]}